### PR TITLE
add upsells to uneditable feature settings

### PR DESCRIPTION
**Pull request by camhashemi**
@cvializ @andrewconner

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/teamSettings/teamSettings.styl [hunk 1]
```diff
@@ -1,5 +1,10 @@
 .kf-team-settings-top-header
   margin-bottom: 15px
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/teamSettings/teamSettings.styl]: if they have the exact same properties, you can combine the two rules like

```
.kf-team-settings-top-header
.kf-team-settings-sub-header
  margin-bottom: 15px
  margin-left: 17px
```

File: kifi-backend/modules/shoebox/angular/src/common/tooltips.styl [hunk 2]
```diff
@@ -32,6 +32,7 @@ tooltip-body()
   transition: transform 0.25s
   width: 150px
   z-index: 10
+  font-smoothing: antialiased
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/common/tooltips.styl:36]: if you're adding it to the tooltip itself, it'd be a good idea to remove it from the `upsell.styl` to get rid of any redundant code

**Comment by cvializ** [comment]
good stuff!

**Comment by camhashemi** [comment]
thanks for the review @cvializ

File: kifi-backend/modules/shoebox/angular/src/teamSettings/teamSettings.js [hunk 3]
```diff
@@ -68,16 +67,16 @@ angular.module('kifi')
...
             ),
             fieldKey: 'view_members',
             selectOptions: getOptions(ORG_SETTING_VALUE.ANYONE, ORG_SETTING_VALUE.MEMBER)
           },
           {
-            title: 'Who can add members to the team?',
+            title: 'Who can invite members to the team?',
             description: (
-              'Select who is able to add members to your team.'
+              'Select who is able to invite members to your team.'
```
Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/teamSettings/teamSettings.js:79]: @cvializ @andrewconner I updated the copy as well (needed for edit_libraries), hopefully they're well received

**Pull request merged**